.github/labeler.yml

```diff
@@ -0,0 +1,93 @@
+# Documentation - https://github.com/actions/labeler
+
+# General Labels
+'Build | Project System':
+  - '.github/*'
+  - '.github/**/*'
```

Comment:
It also includes solution files, cmake and other related stuff.

Reply:
scope for this label expanded